native_client/deepspeech.cc

```diff
@@ -304,23 +304,38 @@ DS_FreeModel(ModelState* ctx)
 }
 
 int
-DS_EnableDecoderWithLM(ModelState* aCtx,
-                       const char* aLMPath,
-                       const char* aTriePath,
-                       float aLMAlpha,
-                       float aLMBeta)
+DS_EnableExternalScorer(ModelState* aCtx,
+                        const char* aScorerPath)
 {
   aCtx->scorer_.reset(new Scorer());
-  int err = aCtx->scorer_->init(aLMAlpha, aLMBeta,
-                                aLMPath ? aLMPath : "",
-                                aTriePath ? aTriePath : "",
-                                aCtx->alphabet_);
+  int err = aCtx->scorer_->init(aScorerPath, aCtx->alphabet_);
```

Comment:
Just as correctly as they were before. Unfortunately KenLM will throw if the path is not a file you can open for reading, or if it's not a valid KenLM binary file format. I'll add an earlier check, since part of this PR is cleaning up the error handling anyway.